xc/xenops_server_xen.ml

```diff
@@ -1199,6 +1199,9 @@ module VM = struct
             begin
               debug "Renaming domain %d from %s to %s" di.Xenctrl.domid vm vm';
               Xenctrl.domain_sethandle xc di.Xenctrl.domid vm';
+              (* TODO: *)
+              let final_uuid_path = "/vm/" ^ vm ^ "/final-uuid" in
+              xs.Xs.rm final_uuid_path;
```

Comment:
I think we should make sure that this doesn't fail if the path does not exist. That would make the `rename` function more generally useful.

Reply:
However as there is already a `safe_rm` function which we add a wrapper for `Xs.rm`, I would like to use it.